lib/data/readme.md

```diff
@@ -0,0 +1,32 @@
+# Introduction
```

Comment:
Can you put your latest readme version into this PR? The file is missing a lot of extra improvements. 🙂

Reply:
Do you mean the information related to other files? Cause apart from that this should be the same.

Reply:
I have not included the other files as this PR only adds `monorepo.json` file. The others will be added in subsequent PRs.